pkg/process/error.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package process
+
+import (
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+func init() {
+	monkit.AddErrorNameHandler(func(err error) (string, bool) {
```

Comment:
Not entirely sure what this error handler adds as a benefit.
Would you explain that for me?

Reply:
Do you want to add additional metrics what grpc error is being returned?

Reply:
i tried to explain this in the PR description - this part:
> when an upload is rejected we return the Unavailable status code over grpc, but monkit doesn't know how to show grpc status codes, so we don't know how many unavailable requests are getting returned. first, add grpc status codes to monkit so monkit knows and can show us.

Reply:
Is that function not run for every err that is being returned by any function?
Not sure about the overhead there.

Reply:
you're absolutely right, but errors (hopefully! ha) don't happen that often.